docs: removes dbt metrics docs (#9012)
lightdash/lightdash: docs/docs/references/metrics.mdx

There are two ways to add metrics to your project in Lightdash:

1. (**Suggested**) [Using the `meta` tag](#using-the-meta-tag)

> 2. [Using dbt's Semantic Layer](#using-dbts-semantic-layer)

Reviewer: Should we add a beta tag here so folks know it's not a GA feature?

> 2. [Using dbt's Semantic Layer](#using-dbts-semantic-layer) (this is still a Beta feature)

Reviewer: Can you update the text in parenthesis to: "(This integration is in public beta in Lightdash)"

> 2. [Using dbt's Semantic Layer](#using-dbts-semantic-layer) (this is still a Beta feature)

Author: ```suggestion
2. [Using dbt's Semantic Layer](#using-dbts-semantic-layer) (this integration is in public beta in Lightdash)
```
